src/effects/effectchainslot.cpp

```diff
@@ -44,6 +44,10 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
     m_pControlChainSuperParameter->set(0.0);
     m_pControlChainSuperParameter->setDefaultValue(0.0);
 
+    m_pControlChainSuperForceUpdate = new ControlObject(ConfigKey(m_group, "super_force_next_update"));
```

Comment:
OK, you set "meta_force_next_update" to 1 and then the next touch of the meta knob will let the effect parameter jump? Or do you have an Init script that updates all Mappings, can the controller do this? 

Why should this only happen for Controllers and not for the GUI? In the current solution the GUI would interfere with the controller. To fix that we need a single call like ForceSet() or something. 

What is the difference between softtakover in the mapping and this one here? 
I think It will ignore the  first control actions after init as well. ...

Thinking about this, It looks like we have two user cases
* Effect setup 
* Effect tweaking as effect 

Dos it make sense to disable softtakover if the effect is disabled?  
This might be coupled with the "Enabled" control that will be off by default (your other PR) 
On the on the hand it might be an unexpected behavior ...

How about introducing a global "enable_softtakover" co switch?  This could be disabled during the initing the controller? 

----

I do not know how fast we will get a conclusion here, so you might want to move the "meta_force_next_update" changes to a separate commit.